lms/static/js/student_account/views/LoginView.js

```diff
@@ -113,6 +113,15 @@
                 this.setErrors();
                 this.element.hide( this.$resetSuccess );
 
+                /* Labster Changes
```

Comment:
Please remove changes in this file to prevent upgrade issues in future. It's ok to have default text `We can't sign you in`